web/js/map/ui.js

```diff
@@ -210,7 +210,7 @@ export default function mapui(models, config, store) {
       case vectorStyleConstants.SET_FILTER_RANGE:
       case vectorStyleConstants.SET_VECTORSTYLE:
       case vectorStyleConstants.CLEAR_VECTORSTYLE:
-      case CALCULATE_RESPONSIVE_STATE:
+      case CALCULATE_SCREEN_SIZE:
```

Comment:
Nothing is dispatching an action with this type so this onResize function to add/remove the scale never gets called when the browser is resized:
![Uploading Screen Shot 2022-09-12 at 9.40.02 AM.png…]()